BackendNew/models/students.js

```diff
@@ -16,7 +16,7 @@ exports.getStudents = callback => {
 exports.getSelectedStudents = (val, callback) => {
   // Sql query for not selected students.
   var notSelectedStudentsSql =
-    "SELECT S.NameWithInitials as Name, S.i as RegNo, B.Name as InternBatch, SC.StudentId FROM Student S, Batch B, Student_Select_Company SC, Company C WHERE S.BatchId = B.BatchId AND S.StudentId = SC.StudentId AND C.CompanyId = SC.CompanyId AND SC.IsSelected = 0 AND S.NameWithInitials NOT IN (SELECT S.NameWithInitials FROM Student S, Batch B, Student_Select_Company SC, Company C WHERE S.BatchId = B.BatchId AND S.StudentId = SC.StudentId AND C.CompanyId = SC.CompanyId AND SC.IsSelected = 1) GROUP BY S.NameWithInitials, S.IndexNumber, B.Name, SC.StudentId";
+    "SELECT S.NameWithInitials as Name, S.IndexNumber as RegNo, B.Name as InternBatch, SC.StudentId FROM Student S, Batch B, Student_Select_Company SC, Company C WHERE S.BatchId = B.BatchId AND S.StudentId = SC.StudentId AND C.CompanyId = SC.CompanyId AND SC.IsSelected = 0 AND S.NameWithInitials NOT IN (SELECT S.NameWithInitials FROM Student S, Batch B, Student_Select_Company SC, Company C WHERE S.BatchId = B.BatchId AND S.StudentId = SC.StudentId AND C.CompanyId = SC.CompanyId AND SC.IsSelected = 1) GROUP BY S.NameWithInitials, S.IndexNumber, B.Name, SC.StudentId";
```

Comment:
Isn't S.IndexNumber and SC.StudentId same?

Reply:
No they are not